doc/model.rst

```diff
@@ -0,0 +1,739 @@
+*****
+Model
+*****
+========
+Overview
+========
+SmartSim ``Model`` objects enable users to execute computational tasks in an
+``Experiment`` workflow, such as launching compiled applications,
+running scripts, or performing general computational operations. ``Models`` can be launched with
+other SmartSim entities and infrastructure to build AI-enabled workflows.
+``Model`` objects can leverage ML capabilities by utilizing the SmartSim client (:ref:`SmartRedis<dead_link>`)
+to transfer data to the ``Orchestrator``, enabling other running SmartSim ``Models`` to access the data.
+Additionally, clients can execute ML models (TF, TF-lite, PyTorch, or ONNX) and TorchScripts stored in the
+``Orchestrator``. SmartRedis is available in Python, C, C++, or Fortran.
+
+To initialize a SmartSim ``Model``, use the ``Experiment.create_model()`` API function.
+When creating a ``Model``, a :ref:`RunSettings<dead_link>` object must be provided. A ``RunSettings``
+object specifies the ``Models`` executable simulation code (e.g. the full path to a compiled binary) as well as
+application execution specifications. These specifications include :ref:`launch<dead_link>` commands (e.g. `srun`, `aprun`, `mpiexec`, etc),
+compute resource requirements, and application command-line arguments.
+
+When initializing a ``Model`` object, users can direct SmartSim to assign values at runtime to variables in the
```

Comment:
Using "application code" is a bit confusing. All we allow with SmartSim is to assign values to parameters in text files, where special tagged keywords are replaced with the values specified by the user. I think maybe this paragraph can be shortened, given that a clear explanation is given in a paragraph below.